scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -635,14 +635,22 @@ def test_single_identity_magnitude():
 def test_identity_invariance():
     n = 10
     p = Rotation.random(n)
-    result = p * Rotation.identity(n) * p.inv()
```

Comment:
This changes the meaning of the test. It is best not to change code unrelated to the PR.

Reply:
It's still testing the invariance right? I added these two tests since I found sometimes p * I * inv(p) is correct but p * I itself is incorrect

Reply:
Well, I sort of agree with @pmla here. 

Let's not involve unrelated changes and discuss them in a separate PR.